packages/eslint-config-ibmdotcom/index.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     'eslint:recommended',
     'plugin:jsx-a11y/recommended',
     'plugin:jsdoc/recommended',
+    'plugin:lit/recommended',
```

Comment:
If we change it to `'plugin:lit/all',` we have more errors:

```
✖ 3172 problems (3170 errors, 2 warnings)
  2870 errors and 2 warnings potentially fixable with the `--fix` option.
```

But that's for another time.